libmscore/harmony.cpp

```diff
@@ -1038,7 +1038,7 @@ void Harmony::layout()
       qreal xx = 0.0;
 
       if (parent()->isFretDiagram())
-            yy = -score()->styleP(Sid::harmonyFretDist);
+            yy = -score()->styleP(Sid::harmonyFretDist) + spatium() * 3.0;
```

Comment:
What is this value representing?

Reply:
This moves the harmony closer down towards the top of the fret diagram, acting against the default harmony offset.

Reply:
So, wouldn't it be better to actually use the default harmony offset, or something calculated from it?  Or really, just let autoplace handle the spacing?

Reply:
I don't know... without this line, the harmony is way too far above the fret diagram. Something calculated from it would be good, I guess.

Reply:
The harmony is too far above because it has the default offset applied to it.  Instead of subtracting that off, better to just make the default offset be something that makes sense.  This can be done by checking for fret diagram in getPropertyStyle() and returning the appropriate Sid.

See my comment with sample code at https://github.com/musescore/MuseScore/pull/5029#issuecomment-494212988.

The one thing I didn't consider is that the offset is calculated from the *position* of the fret diagram, not the *top* of it, sp the fretMinDistance value isn't actually enough.  You would need to add fd->bbox().top(0 to yy, I think.